<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>РК 2.2 | ДГИ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Подключение</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2" colspan="2">
          <widget class="QComboBox" name="com_combo">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Последовательный порт</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="com_button">
           <property name="text">
            <string>Подключить</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="net_button">
           <property name="text">
            <string>Открыть</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="port2_sb">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>5601</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="5">
          <widget class="QTextEdit" name="log_edit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Порты TCP/IP</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="port1_sb">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>5600</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="refreshPorts_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Поток данных:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLabel" name="comSpeed_label">
           <property name="text">
            <string>0 Б/с</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="resetRotation_button">
           <property name="text">
            <string>Сброс вращения</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Сырые данные</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="x_sb">
           <property name="prefix">
            <string>ax = </string>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="y_sb">
           <property name="prefix">
            <string>ay = </string>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="z_sb">
           <property name="prefix">
            <string>az = </string>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
           <property name="value">
            <double>-60.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="gx_sb">
           <property name="prefix">
            <string>gx = </string>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="gy_sb">
           <property name="prefix">
            <string>gy = </string>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="gz_sb">
           <property name="prefix">
            <string>gz = </string>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="T_sb">
           <property name="prefix">
            <string>T = </string>
           </property>
           <property name="suffix">
            <string> °C</string>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>22.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>79</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Обработка</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="pitch_sb">
           <property name="prefix">
            <string>pitch = </string>
           </property>
           <property name="suffix">
            <string> deg</string>
           </property>
           <property name="minimum">
            <double>-999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="roll_sb">
           <property name="prefix">
            <string>roll = </string>
           </property>
           <property name="suffix">
            <string> deg</string>
           </property>
           <property name="minimum">
            <double>-999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="px_sb">
           <property name="prefix">
            <string>x = </string>
           </property>
           <property name="minimum">
            <number>-9999</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="py_sb">
           <property name="prefix">
            <string>y = </string>
           </property>
           <property name="minimum">
            <number>-9999</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Визуализация</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="GLWidget" name="widget" native="true">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PointWidget" name="pointWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Гироскоп</string>
    </property>
    <addaction name="calibrateZero_action"/>
    <addaction name="calibrateAngle_action"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="about_action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="calibrateZero_action">
   <property name="text">
    <string>Калибровка нуля</string>
   </property>
   <property name="toolTip">
    <string>Калибровка нуля</string>
   </property>
  </action>
  <action name="calibrateAngle_action">
   <property name="text">
    <string>Калибровка угла</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PointWidget</class>
   <extends>QWidget</extends>
   <header>pointwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
